Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -284,6 +456,17 @@ YUI.add('ez-subitemlistview', function (Y) {
              * @type Array of {Object} array containing location structs
              */
             subitems: {},
+
+            /**
+             * Boolean to check if a subitem priority is currently selected.
+             *
+             * @attribute isOnePrioritySelected
+             * @default false
+             * @type Boolean
+             */
+            isOnePrioritySelected: {
```

Comment:
the question in https://github.com/ezsystems/PlatformUIBundle/pull/527#discussion_r55807310 is still pending even if you change something while I'm reviewing.